js/components/time.js

```diff
@@ -92,27 +92,12 @@ Fliplet.FormBuilder.field('time', {
     }
 
     if (!this.value || this.autofill === 'always') {
-      var now = new Date();
-      var hours = now.getHours();
-      var minutes = now.getMinutes();
-
-      if (hours < 10) {
-        hours = '0' + hours;
-      }
-
-      if (minutes < 10) {
-        minutes = '0' + minutes;
-      }
-
-      this.updateValue(hours + ':' + minutes);
+      this.updateValue(moment().format('HH:mm'));
       this.empty = false;
     }
-  },
-  watch: {
-    value: function(val) {
-      if (!val) {
-        this.updateValue(moment().format('HH:mm'));
-      }
+
+    if (!this.value) {
+      this.updateValue(moment().format('HH:mm'));
```

Comment:
It looks like if `this.value` is falsey, it goes through `if (!this.value || this.autofill === 'always') { this.updateValue(moment().format('HH:mm')); this.empty = false; }` once and then `this.updateValue(moment().format('HH:mm'));` again.

Something seems redundant here. Can you explain it had to be coded it this way?

Reply:
> It looks like if `this.value` is falsey, it goes through `if (!this.value || this.autofill === 'always') { this.updateValue(moment().format('HH:mm')); this.empty = false; }` once and then `this.updateValue(moment().format('HH:mm'));` again.
> 
> Something seems redundant here. Can you explain it had to be coded it this way?

you are right, I corrected the solution and removed the extra code